testdata/jsight_0.3/req.japi.annotations/allow_enum.json

```diff
@@ -0,0 +1,54 @@
+{
+	"tags": {},
+	"userEnums": {
+		"@animalEnum": {
+			"annotation": "annotation",
+			"description": "",
+			"value": {
+				"key": "@animalEnum",
```

Comment:
Thanks to this example, I found an error in the data schema. I'm correcting it.
However, I think we should delete the "key".

Reply:
Done